lib/fastlane/plugin/firebase_app_distribution/actions/firebase_app_distribution_remove_testers_action.rb

```diff
@@ -26,9 +26,13 @@ def self.run(params)
           UI.user_error!("A maximum of 1000 testers can be removed at a time.")
         end
 
-        UI.message("⏳ Removing #{emails.count} testers from project #{params[:project_number]}...")
-
-        count = fad_api_client.remove_testers(params[:project_number], emails)
+        if blank?(params[:group_alias])
+          UI.message("⏳ Removing #{emails.count} testers from project #{params[:project_number]}...")
+          count = fad_api_client.remove_testers(params[:project_number], emails)
+        else
+          UI.message("⏳ Removing testers from group #{params[:group_alias]}...")
```

Comment:
Consider specifying the count of testers in the message like in the case of removing testers from a project.

Reply:
Done.